codecov.yml

```diff
@@ -20,5 +20,6 @@ coverage:
     - "tuf/testutils/*"
     - "vendor/*"
     - "proto/*.pb.go"
+    - "trustmanager/remoteks/keystore.proto"
```

Comment:
Ok fixed now. Must have been one of those occasions where I was thinking of something else while typing and the wrong thing came out.